src/components/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+
+import { Todo } from '../types/Todo';
+import { FilterOption } from '../types/FilterOptions';
+import { useTodo } from '../hooks/useTodo';
+
+type Props = {
+  activeTodos: Todo[],
+  filter: string,
+  setFilter: (filter: string) => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  activeTodos,
+  filter,
+  setFilter,
+}) => {
+  const { todos, deleteTodoHandler } = useTodo();
+
+  const completedTodos = useMemo(() => {
+    return todos.filter(({ completed }) => completed === true);
+  }, [todos]);
```

Comment:
```suggestion
  const completedTodos = todos.filter(({ completed }) => completed);
```

useMemo is useless here